menas/src/main/scala/za/co/absa/enceladus/menas/services/SchemaRegistryService.scala

```diff
@@ -69,8 +120,34 @@ class SchemaRegistryService @Autowired()() {
    */
   def loadSchemaByUrl(remoteUrl: String): SchemaResponse = {
     try {
+      if (keyManagerFactory.isEmpty) {
+        logger.warn("keyManagerFactory is not defined, secure schema registry integration may not function properly. " +
```

Comment:
It's a logging within a web app backend. Will be hard to spot for the app owner. 🤔

Reply:
Note: If cannot retrieve the schema, try to let the user know what went wrong (http return code?)

Reply:
After some digging, it turns out that the http return code is only available as part of the _cause string_ and only in some of the use-cases. So exposing the cause string seems like the best option for the user, it actually dramatically improves the level of understanding what the problem might be:

![keyStoreNotFound](https://user-images.githubusercontent.com/4457378/96007512-d4730100-0e3e-11eb-98de-5bb93032cd1e.png)
![KeyStorePwdError](https://user-images.githubusercontent.com/4457378/96007526-d5a42e00-0e3e-11eb-9334-a5fb73e166a0.png)
![ssr403](https://user-images.githubusercontent.com/4457378/96007528-d5a42e00-0e3e-11eb-9ad7-64f422513fb7.png)
![ssrHostException](https://user-images.githubusercontent.com/4457378/96007533-d63cc480-0e3e-11eb-9bd3-1d81c9d4a050.png)

Reply:
Wondering it it won't disclose some sensitive information or not too "programmer's language" for UI.
But let's leave it like this for now 👍

Reply:
To the extent I tested it and thought about it, it looked reasonable.